templates/components/search/display_data.html.twig

```diff
@@ -58,6 +58,22 @@
                {% endif %}
             {% else %}
                {{ include('components/search/controls.html.twig') }}
+               {% if normalized_itemtype == 'Plugin' %}
+                  {% set url = call('Plugin::getFormURL', []) %}
+                  {% set config_value = call('Config::getConfigurationValues', ['core', ['marketplace_replace_plugins']]) %}
+                  {% if config_value['marketplace_replace_plugins'] != '1' %}
+                     {% if config_value['marketplace_replace_plugins'] == '3' %}
```

Comment:
```suggestion
                  {% set config_value = call('Config::getConfigurationValue', ['core', 'marketplace_replace_plugins']) %}
                  {% if config_value is not null and config_value != constant('Glpi\\Marketplace\\Controller::MP_REPLACE_ASK')  %}
                     {% if config_value is not null and  config_value == constant('Glpi\\Marketplace\\Controller::MP_REPLACE_NEVER')  %}
```

Reply:
Use the constant to check value

e.g : ```Controller::MP_REPLACE_YES```

Use ```getConfigurationValue``` instead of  ```getConfigurationValues```  for direct access to the database value or null